deployer/src/proxy.rs

```diff
@@ -74,7 +75,7 @@ pub async fn handle(
     };
 
     // Record current service for tracing purposes
-    span.record("service", &service);
+    span.record("shuttle.service", &service);
```

Comment:
`service` is already used to designated the Shuttle service (logger, gateway, ..). Might want to even rename it to something like `user.service`. But this is why it was renamed.